source/reference/footnotes.txt

```diff
@@ -0,0 +1,54 @@
+=========
+Footnotes
+=========
+
+Use footnotes to add detail that not all readers will need.
+
+Syntax
+------
+.. procedure::
+    :style: normal
+
+    .. step:: Mark the footnote location in your text with ``[#footnote-identifier]``,
+        where "footnote-identifier" is arbitrary. 
+
+    .. step:: If the bottom of the page doesn't have a "Footnotes" rubric, use 
+        ``.. rubric:: Footnotes`` to create one.
+    
+    .. step:: Beneath the "Footnotes" rubric, enter  ``.. [#footnote-identifier] 
+                <footnote text>`` to add your footnote.
+
```

Comment:
Since this describes the steps in how to create one, lets put a codeblock to show what these steps describe.